doc/releases/general.rst

```diff
@@ -122,7 +122,7 @@ Release timeline
 .. ceph_timeline:: releases.yml development nautilus mimic luminous kraken jewel infernalis hammer giant firefly emperor
 
 .. _Octopus: ../octopus
-.. _15.1.0: ../octopus#v15-1-0-octopus
+.. _15.2.0: ../octopus#v15-2-0-octopus
```

Comment:
either we need a `15.1.1` link here or we should add `skip_ref: true` to 15.1.1 in `releases.yml`